### DISCUSSION TITLE
What would AccountManager have provided?

### DISCUSSION THREAD

**Discussion by beedell-roke** [Chesire/Nekome#999] [Q&A] [answered]
Per https://github.com/Chesire/Nekome/issues/432#issue-775600044, what functionality would this project have gained – multiple account support?

**Comment by Chesire** [comment, marked as answer]
The main idea for using this was to use the correct account storage mechanism for Android, as opposed to manually storing tokens. There shouldn't be any issue with using multiple accounts with the current implementation, other than needing to add that functionality
